spec/dependent_styles_spec.rb

```diff
@@ -1,82 +1,79 @@
-Dependents.each_pair do |id, (filename, path, style, reason)|
+Dependents.each_pair do |basename, (filename, path, style, reason)|
 
-  describe "dependent style #{id}" do
+  describe "#{basename} (dependent):" do
 
-    it "is a valid CSL 1.0 style" do
+    it "must validate against the CSL 1.0.1 schema" do
       expect(CSL.validate(path)).to eq([])
     end
 
-    it "has a conventional file name" do
+    it "must have a conventional file name" do
       expect(filename).to match(/^[a-z\d]+(-[a-z\d]+)*\.csl$/)
     end
 
-    it "was successfully parsed" do
+    it "must be parsable as a CSL style" do
       expect(style).to be_a(CSL::Style), reason
     end
 
     unless style.nil?
-      it "is dependent" do
+      it 'only dependent styles are allowed in the "dependent" subdirectory' do
         expect(style).to be_dependent
       end
 
-      it "has an info element" do
+      it "must contain an <info/> element" do
        expect(style).to have_info
       end
 
-      it "does not have any rendering elements" do
+      it '"dependent" styles may not contain <macro/>, <citation/>, or <bibliography/> elements' do
```

Comment:
Again, I'd like to be a little bit more wordy here in case the contributor misplaced an independent style.